containers/README.md

```diff
@@ -4,3 +4,18 @@ Container image gets automatically built by quay.io at [Cerberus image](https://
 
 ### Run containerized version
 Refer to the [instructions](https://github.com/openshift-scale/cerberus#Run-containerized-version) for information on how to build and run the containerized version of cerberus.
+
+### Cerberus as a KubeApp
+To run containerized Cerberus as a Kubernetes/OpenShift Deployment, follow these steps:
+1. Configure the [config.yaml](https://github.com/openshift-scale/cerberus/tree/master/config) file according to your requirements.
+2. Switch to default namespace:
+    - In Kubernetes, use `kubectl config set-context --current --namespace=default`
+    - In OpenShift, use `oc project default`
+3. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>`
+4. Create a ConfigMap named cerberus-config using `kubectl create configmap cerberus-config --from-file=<path_to_cerberus_config>`
+5. Create a Deployment and a NodePort Service using `kubectl apply -f cerberus.yml`
+6. Accessing the go/no-go signal:
+    - In Kubernetes, use `<host_IP_address>:30000` to access the signal.
```

Comment:
Let's use kubectl port-forward to access the go/no-go signal on local ports.

Reply:
Okay